conversations/v3/whatsapp.yaml

```diff
@@ -3098,7 +3096,7 @@ components:
             IMAGE
         text:
           type: string
-          description: Text based part of format "TEXT
+          description: Text part in the case the format is TEXT
```

Comment:
`Text based part of format "TEXT` sound bit weird to me. Hopefully, my suggestion is somewhat better. @ridlees suggests `text value for the TEXT format.`. Feel free to ignore both of us :-)

Reply:
that's better :) thanks